ONSdigital/dp-download-service
# Force authentication when running in publishing.

* Update to use latest go-ns identity middleware
* Update to latest  go-ns dataset API client
* Only enable the identity check when running in publishing
* Remove a few unused vars
* Add `go:generate` directives for mock gen
* Remove redundant secret key while I am in here

potential todo: update to latest go-ns filter API client.
